Configuration/StandardSequences/python/Eras.py

```diff
@@ -60,7 +60,7 @@ def __init__(self):
         # Scenarios further afield.
         # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
         # include phase 1 stuff until phase 2 tracking is fully defined....
-        self.Phase2 = cms.ModifierChain( self.phase1Pixel, self.trackingPhase1, self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
+        self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
```

Comment:
@lgray 
I missed the context of phase1Pixel and trackingPhase1 added (in part also that it was added by you in part for this test to run). Thanks for pointing this out.
I'm not sure what's the best at this point: maybe phase1Pixel is still meaningful, but then trackingPhase1 doesn't make sense. So, removal should be OK.

@boudoul @makortel @ebrondol  please confirm that 
```python
self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
```
is what's expected from tracker/tracking perspective